rubin_sim/phot_utils/photometric_parameters.py

```diff
@@ -148,76 +152,55 @@ def __init__(
-            electons per pixel per second (defaults to LSST value)
-
-        Parameters
-        ----------
-        othernoise : `Unknown`
-            electrons per pixel per exposure (defaults to LSST value)
-
-        Parameters
-        ----------
-        platescale : `Unknown`
-            arcseconds per pixel (defaults to LSST value)
-
-        Parameters
-        ----------
-        sigma_sys : `Unknown`
-            systematic error in magnitudes
-            (defaults to LSST value)
-
-        Parameters
-        ----------
-        bandpass : `Unknown`
-            is the name of the bandpass to which these parameters
-            correspond.  If set to an LSST bandpass, the constructor will initialize
-            PhotometricParameters to LSST default values for that bandpass, excepting
-            any parameters that have been set by hand, i.e
-
-        myPhotParams = PhotometricParameters(nexp=3, bandpass='u')
-
-        will initialize a PhotometricParameters object to u bandpass defaults, except
-        with 3 exposures instead of 2.
-
-        If bandpass is left as None, other parameters will default to LSST r band
-        values (except for those values set by hand).  The bandpass member variable
-        of PhotometricParameters will, however, remain None.
+        exptime : `float`
+            Exposure time in seconds (per exposure).
+            None will default to value from DefaultPhotometricParameters.
+        nexp : `int`
+            Number of exposures per visit.
+            None will default to value from DefaultPhotometricParameters.
+        effarea : `float`
+            Effective area in cm^2.
+            None will default to value from DefaultPhotometricParameters.
+        gain : `float`
+            Electrons per ADU.
+            None will default to value from DefaultPhotometricParameters.
+        readnoise : `float`
+            Electrons per pixel per exposure.
+            None will default to value from DefaultPhotometricParameters.
+        darkcurrent : `float`
+            Electons per pixel per second.
+            None will default to value from DefaultPhotometricParameters.
+        othernoise : `float`
+            Electrons per pixel per exposure.
+            None will default to value from DefaultPhotometricParameters.
+        platescale : `float`
+            Arcseconds per pixel.
+            None will default to value from DefaultPhotometricParameters.
+        sigma_sys : `float`
+            Systematic error in magnitudes.
+            None will default to value from DefaultPhotometricParameters.
+        bandpass : `str`
+            The name of the bandpass for these parameters.
+            If set to an LSST bandpass, the constructor will initialize
+            PhotometricParameters to LSST default values for that bandpass,
+            excepting any parameters that have been set by hand.  e.g.
+
+        >>> myPhotParams = PhotometricParameters(nexp=3, bandpass='u')
```

Comment:
Maybe this part should go in an "Examples" section.

Reply:
Could see this going in multiple options, given that it's a description in a parameter. But it's also unwieldy in a parameter description, so updated.